lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -36,7 +36,7 @@
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 28
+LIBPATCH = 29
```

Comment:
Hi, the library got updated since, please sync with main and bump libpatch to 30

Reply:
I see @marceloneppel fixed this issues